pkg/webhooks/server.go

```diff
@@ -292,6 +292,18 @@ func writeResponse(rw http.ResponseWriter, admissionReview *v1beta1.AdmissionRev
 func (ws *WebhookServer) resourceMutation(request *v1beta1.AdmissionRequest) *v1beta1.AdmissionResponse {
 	logger := ws.log.WithName("MutateWebhook").WithValues("uid", request.UID, "kind", request.Kind.Kind, "namespace", request.Namespace, "name", request.Name, "operation", request.Operation, "gvk", request.Kind.String())
 
+	if request.Operation == v1beta1.Delete {
```

Comment:
I believe this should be below `if excludeKyvernoResources(request.Kind.Kind) {`